### ISSUE TITLE
[BUG] Big docker images (>1GB) make the process "not responding"

### ISSUE THREAD

**Issue by bostrot** [bostrot/wsl2-distro-manager#175]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Creating a distro from a big docker image makes the app unresponding while it is being created. (Probably in the re-packing of the archive) It still works but is not the best user experience.

### Environment

```markdown
- Windows Version:
- WSL Distro Manager Version: v1.8.4
- WSL Version: 
- Source:
```

**Comment by PiotrEsse** [comment]
In my case creating instance fron dockerhub took very long (AMD 5800, 64GB Ram, SSD). Addres from dockerhub dockerhub:garystafford/langflow:0.3.3.

After download an image WSL manager hang for about 5-8 min without any message. Usually users will shutoff application but I was trying to check it will finish installing instance and after that time app just become responsive and docker image was installed and visible on the installed instances.

I would expect to have an info about present state, some king of progress bar. Now users will mistakenly thing that app crashed.


WSL manager 1.8.8. Win11 PRO

**Comment by ifzm** [comment]
Download progress is need )

**Comment by bostrot** [comment]
Should be fixed by v1.8.15
